doc/gettingStarted/Applications.md

```diff
@@ -0,0 +1,102 @@
+# Applications
+
+InfiniTime has 13 apps on the `main` branch at the time of writing.
+
+## List of apps
+- Stopwatch
+- Alarm
+- Timer
+- Steps
+- Heartrate
+- Music
+- InfiniPaint
+- Pong
+- 2048
+- Dice
+- Metronome
+- Maps
+- Weather
+
+### Stopwatch
+![Stopwatch UI](/doc/gettingStarted/AppsScreenshots/stopwatch.png)
+- Press the Start button (bottom right) to start or stop the timer.
+   - You can also press the side button while the timer is running to pause the timer.
+- Press the Flag button (bottom left) to add a lap.
+- The stopwatch will not yet contimue counting time while the app is closed.
+
+### Alarm
+![Alarm UI](/doc/gettingStarted/AppsScreenshots/alarm.png)
+- Ajust the time with the time picker.
+- Press the Info button in the top middle to see time remaning.
+- Use the toggle in the bottom left to turn the alarm on/off.
+- Use the button in the bottom right to change the alarm days.
+   - You can pick from once, daily, or Monday-Friday.
+InfiniPaint
+### Timer
+![Timer UI](/doc/gettingStarted/AppsScreenshots/timer.png)
+- Ajust how long the timer should go for with the time picker.
+- Press the Start button at the bottom to start/stop the timer.
+
+### Steps
+![Steps UI](/doc/gettingStarted/AppsScreenshots/steps.png)
+- The total count of steps for the current display will show in the middle of the screen.
+- The Reset button in the bottom middle resets the Trip counter. (Total of all steps taken.)
+- The progress circle shows the percentage of your daily goal completed.
+
+### Heartrate
+![Heartrate UI](/doc/gettingStarted/AppsScreenshots/Heartrate.png)
+- Press Start to start measuring your heartrate.
+    - It may take a bit to get the first measurement.
+    - The monitoring currently does not happen while your screen is off, however the screen will not go to sleep while the heartrate app is open.
+
+### Music
+![Music UI](/doc/gettingStarted/AppsScreenshots/Music.png)
+- This app shows currently playing music.
+    - Please note that this app is not very useful without a device connected.
+- Press the button in the center to play/pause, and the buttons on the left and right to go to the previous and next tracks, respectively.
+- Swipe up to get to volume controls.
+
+### InfiniPaint
+![InfiniPaint UI](/doc/gettingStarted/AppsScreenshots/InfiniPaint.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Draw on the screen to add lines.
+- Hold down in one spot to change paint colors.
+
+### Pong
+![Pong UI](/doc/gettingStarted/AppsScreenshots/Pong.png)
+- This app does not allow you to swipe from the top to exit, use the side button instead.
+- Drag your finger to move the paddle.
+- Goal: Don't let the ball go off the left side of the screen.
+
+### 2048
```

Comment:
The game is called "2" on the watch (I would assume for copyright reasons?) so we should probably name it accordingly here

Reply:
Done!